src/sentry/workflow_engine/endpoints/validators/base/detector.py

```diff
@@ -81,9 +81,7 @@ def create(self, validated_data):
                 request=self.context["request"],
                 organization=self.context["organization"],
                 target_object=detector.id,
-                event=audit_log.get_event_id(
-                    "UPTIME_MONITOR_ADD"
-                ),  # TODO this is the wrong event type
```

Comment:
I think someone put "uptime monitor add" here as a placeholder to be sure we create some type of audit log entry here, but it's clearly the wrong type.